test/semantics/stop01.f90

```diff
@@ -18,11 +18,11 @@ program main
   integer, pointer :: p_i
   integer(kind = 1) :: invalid = 0
   integer, dimension(1:100) :: iarray
-  integer, dimension(1:100), pointer :: p_iarray
-  integer, allocatable, dimension(1:100) :: aiarray
+  integer, dimension(:), pointer :: p_iarray
```

Comment:
I notice that this file has an ARM copyright.  Should we be adding NVIDIA to it?

Reply:
Good question for @sscalpone. In this case there is not a lot of new IP going in so I'm not going to worry about it.

Reply:
NVIDIA policy is to add copyright if the change is substantial, a contribution worthy of recognition of ownership.